site/content/docs/include/publish.en.md

```diff
@@ -9,7 +9,7 @@ publish:
     - name: order-events
 ```
 
-In the example above, this manifest declares an SNS topic named `order-events` that other worker services which are deployed to the Copilot environment can subscribe to.
+In the example above, this manifest declares an SNS topic named `order-events` that other worker services which are deployed to the Copilot environment can subscribe to. An environment variable named `COPILOT_SNS_TOPIC_ARNS` is injected into your workload as a JSON string. For this example, you could write: `const {order-events} = JSON.parse(process.env.COPILOT_SNS_TOPIC_ARNS)` in JavaScript. For more detail, see the [pub/sub](../developing/publish-subscribe.en.md) page.
```

Comment:
How about we make `const {order-events} = JSON.parse(process.env.COPILOT_SNS_TOPIC_ARNS)` a code block instead of inline? Just feel like it would be more readable...

Reply:
Does this actually work `order-events` can you have a dash in the variable name? 😮 

Maybe we can also add a short line with a generic format too:
```json
{
  "firstTopicName": "arn:aws:sns:us-east-2:123456789012:firstTopic",
   "secondTopicName": "arn:aws:sns:us-east-2:123456789012:secondTopic",
}
```

Reply:
sorry my comment wasn't clear! Hope the screenshot below makes it clearer:

![Screen Shot 2021-12-01 at 8 44 13 AM](https://user-images.githubusercontent.com/879348/144276264-1996bc45-3e52-454b-b111-e29e2de64de6.png)

`order-events` as a SNS topic name is fine but users can't write the JS code above without an error since dashes in variable names aren't allowed. So my suggestion was to pick a sample topic name that doesn't have a dash or do:
```js
const topic = JSON.parse(process.env.COPILOT_SNS_TOPIC_ARNS)['order-events']
```